### DISCUSSION TITLE
ETA for Quartz 3.2

### DISCUSSION THREAD

**Discussion by mdmoura** [quartznet/quartznet#962] [General] [answered]
Any ETA for Quartz 3.2 so it. can be used with a Hosted Services application?

**Comment by lahma** [comment, marked as answer]
You can already have that with https://www.nuget.org/packages/Quartz.AspNetCore package, it will just move to more specific package in 3.2. 3.2 preview builds are available on MyGet and the 3.2 will be out hopefully at the end of the month, there's some confusion with package id ownership that needs to be sorted.

**Reply by mdmoura** [reply]
I am asking because I created a Worker application with ASP.Net Core 3.1 using the CLI:
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#web-others

Then I installed Quartz.AspNetCore 3.1.0 and added the following code to application's Program class:
https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/hosted-services-integration.html#using

But the extension `AddQuartzHostedService` is not recognised. Why is that?

That is why I though integration with Hosting Services would be available only in the future.

**Reply by lahma** [reply]
You should [follow this guidance](https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/aspnet-core-integration.html#installation). So using the more broad AspNetCore package is the way in 3.1 for now.

**Reply by mdmoura** [reply]
Thank you. Just made it work

**Reply by lahma** [reply]
Quartz 3.2 has been released.
